tensorflow/compiler/xla/service/gpu/gpu_compiler.cc

```diff
@@ -904,7 +904,7 @@ static Status CompileModuleToLlvmIrImpl(
     const std::string& platform_name, const se::Platform::Id platform_id,
     GpuDeviceInfo gpu_device_info,
     se::CudaComputeCapability cuda_compute_capability,
```

Comment:
This is better than what we've had, thanks!

WDYT about passing around GpuVersion defined below instead? Could also make it a custom wrapping struct with shared methods (up to you?) Also maybe GpuComputeCapability could be a better name (again up to you)

Reply:
OK!

Reply:
Agree, "GpuComputeCapability" would be nice, but this PR is a good step in that direction without having to change everything at once.